### ISSUE TITLE
[White Label-Card Bingsports]: Narrator does not navigate Card controls when using arrow keys.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5718]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users cannot explore the contents of a card using arrow navigation, forcing them to rely only on Tab. This results in slower, more cumbersome navigation and makes it difficult to understand the logical structure of the content.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Card Bingsports word in type your message text box.
4. Response will appear.
5. Navigate to the cards in the response.
6. Observe an issue that Narrator does not navigate Card controls when using arrow keys. 

### What do you expect?

The element should expose correct programmatic structure and relationships so Narrator can navigate through the Card controls using arrow keys. 

### What actually happened?

Narrator does not navigate Card controls when using arrow keys. 

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

### Adaptive Card JSON

```json

```
